packages/kmod-6.12-nvidia-r570/grid-license-check.service

```diff
@@ -0,0 +1,26 @@
+[Unit]
+Description=GRID License Check
+RefuseManualStart=true
+RefuseManualStop=true
+DefaultDependencies=no
+Before=kubelet.service
+After=nvidia-gridd.service
+Requires=nvidia-gridd.service
+
+[Service]
+Type=oneshot
+ExecCondition=/usr/bin/ghostdog match-nvidia-driver grid
+# Otherwise, attempt to load the module.
+ExecStart=/usr/bin/nvidia-smi -q
+# Ensure that the stderr file exists. Otherwise, grep fails on an empty file.
```

Comment:
It's the STDOUT file what you are creating, right?

Reply:
Yes, this is a forgotten update, I moved to STDOUT but forgot to update the comment.